app/views/foods/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2>
+  Here's a list of your foods!
+</h2>
+<%= link_to(foods_path) do %>
```

Comment:
- [ ] [REQUIRED] foods path is the current path it should leed to where new food is added. use  `new_food_path` instead